lib/schemas/fastly.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+function isSecure (req) {
+  return ~['1', 'true'].indexOf(req.headers['fastly-ssl'])
```

Comment:
It seems according to the docs at https://docs.fastly.com/guides/backend-servers/maintaining-separate-http-and-https-requests-to-backend-servers just the existence of the header with any value is enough to signal SSL. Is there a reason to check the value here?